stratum/hal/lib/tdi/es2k/BUILD

```diff
@@ -0,0 +1,226 @@
+# stratum/hal/lib/tdi/es2k/BUILD
+
+# Copyright 2018 Google LLC
+# Copyright 2018-present Open Networking Foundation
+# Copyright 2022 Intel Corporation
```

Comment:
I see 3 variations in es2k files:
2022 Intel
2023 Intel
2022-2023 Intel

Reply:
This assortment of dates is not unreasonable. You generated the original set of files in 2022. Some of those were modified, and others were added, in 2023. The range is supposed to indicate the years in which IP content was _added_ to the files, so the date would remain 2022 if no changes were made in 2023.